### PR TITLE
DT-1595 AppInsights and plugin upgrade

### PR DESCRIPTION
**Pull request by andymarke**

### CODE REVIEW

File: helm_deploy/dps-smoketest/templates/_envs.tpl [hunk 1]
```diff
@@ -43,4 +43,8 @@ env:
         name: {{ template "app.name" . }}
         key: APPINSIGHTS_INSTRUMENTATIONKEY
 
+  - name: APPLICATIONINSIGHTS_CONNECTION_STRING
```
Comment (petergphillips) [inline helm_deploy/dps-smoketest/templates/_envs.tpl:46]: the name above needs changing from `APPLICATION_INSIGHTS_IKEY` to `APPINSIGHTS_INSTRUMENTATIONKEY`

Comment (andymarke) [reply helm_deploy/dps-smoketest/templates/_envs.tpl:46]: Done

**Review by petergphillips — APPROVED** [review]
LGTM

File: build.gradle.kts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 plugins {
-  id("uk.gov.justice.hmpps.gradle-spring-boot") version "2.1.2"
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "3.1.0"
   kotlin("plugin.spring") version "1.4.21"
```
Comment (petergphillips) [inline build.gradle.kts:3]: this can go to 1.4.30

**Pull request merged**